corehq/apps/cloudcare/templates/formplayer/query_view.html

```diff
@@ -3,7 +3,10 @@
 
 <script type="text/template" id="query-view-list-template">
   <form>
-    <p><h2 tabindex="0"><%- title %></h2></p>
```

Comment:
Not clear to me what the `<p>` tag was giving us here, so I'm curious about your reasons for removing it (and for using a `<p>` for `description`). Did it have a noticeable affect?

Reply:
It seemed like the intent was maybe to ensure a line break on either side of the heading (which displays on its own line by default), but the actual rendered html put an entire new paragraph on either side of the heading to try to 'fix' the incorrect nesting. So it would come out as:
```html
<p></p>
<h2>(title)</h2>
<p></p>
```